PERFORMANCE.md

```diff
@@ -84,7 +84,7 @@ When a PR is created and the performance tests are needed, if you are a member o
 [Developers](https://github.com/orgs/kroxylicious/teams/developers), you may add the following comment into the PR to trigger the run.
 
 ```
-@tealc-ci run perf
+@kroxylicious-robot run perf
```

Comment:
not directly related to this, but the difference between `run perf` and `run system tests` always bugs me. It should probably be `run perf tests` or maybe `run perf 02-no-filters`  `run system [operator] [proxy]` where we can add specific groups